#+TITLE:Notes
#+AUTHOR: Jackson

* Chapter 2
** DONE add dates from syllabus to calendar
** Grading
*** Exams 80%
*** labs 20%
** Introduction and Chemical Biology
| Element | Symbol |
|---------+--------|
| H       |    63% |
| O       |     26 |
| C       |      9 |
| N       |      1 |
- atoms are made up of protons, neutrons, and electrons
- we only care about outer electrons
*** Molecule vs Compound
- two or more atoms vs two or more different elements
*** Types of bonding
Covalent bonds: electrons are shared among atoms
- polar: unequal sharing
- non polar: equal sharing
- single and double covalent bonds
  - depends on how many electrons are being shared
- Polar examples
  - Oxygen and hydrogen can form a covalent bond
    - water is a polar substance
  - Sulfur and hydrogen can also
  - Same with N and H
- Non polar examples
  - C and H
  - C and C
Ionic bonds: One or more electrons from one atom are removed and attached to another atom, forms cations and anions
- this will make both atoms "happy", or closer to a completed valence shell
  - These can also be called salts
- Examples
  - Na Cl, Na has one, Cl has 7, they can combine and make each complete
Hydrogen bonds: Weak bonds between hydrogen and other atoms
- Water is polar and it makes a ton of hydrogen bonds by orienting its O to another water molecules H (weak Hydrogen bond)
| Chemical Atom | Symbol | Ion           | Electrons Gained or Lost |
|---------------+--------+---------------+--------------------------|
| Hydrogen      | H      | Hydrogen Ion  | 1 Lost                   |
| Sodium        | Na     | Sodium Ion    | 1 Lost                   |
| Potassium     | K      | Potassium Ion | 1 Lost                   |
| Chlorine      | Cl     | Chloride Ion  | 1 Gained                 |
| Magnesium     | Mg     | Magnesium Ion | 2 Lost                   |
| Calcium       | Ca     | Calcium Ion   | 2 Lost                   |
*** Electronegativity
- when two atoms with differing electronegativity bond the electrons will concentrate more towards the atom with greater electronegativity (polar covalent bonding)
  - if two atoms with the same electronegativity bond they will form a non polar bond
  - non polar bonds are hydrophobic
  - polar bonds are hydrophilic
  - polar dissolves polar
*** Ionic Molecules
- Ionization
  - carboxyl group
    - (-COOH)
  - amino group
    - (-NH2)
*** Fucntion Groups
- Ionized Group
  - Carboxyl
  - Amino
  - Phosphate
** Free Radicals
- atoms of molecules containing a single electron
- Unstable, and highly reactive
** Solutions
- the liquid everything is disolved in is the solvent
- Everything else is the solute
- water is a universal solvent
- however not all molecules can be dissolved in water
*** rxn with water
- hydrolysis breaking of a chemical bond with the additions of elements of water -H and -OH to the products
- dehydration involves a removal of water, one net water molecules is removed to combine two small molecules into one larger one
** Osmosis
- water moving between fluid compartments
- water moves from areas of low concentrations of solutes, to areas of high solute concentration, essentially creating two equal areas of solute concentration
- rate of diffusion: 10 billion molecules per second
** Solubility in water
- polar molecules will easily dissolve in water: hydrophilic
- non polar molecules will not easily dissolve in water: hydrophobic
** Amphipathic Molecules
- a special class of molecules that have a polar or ionized region at one site and a nonpolar region at another site
- in water these molecules will form clusters with their polar regions at the surface of the cluster, and the non polar sites inwards
- these help dissolve non polar substances in the presense of water
- plasma membrane structure helps transport molecules in the blood
** Concentration
- the amount of solute present in a unit volume of solution
- moles/liter is an example
** Acids and Bases
- molecules that release H are called acids
- mocules that accept H are called bases
- Hydrogen is very useful in our bodys energy system due to the fact that it is very simple
- The bodies pH range is around 7.35 to 7.45
  - blood 7.4
** Terminology of different Sciences
- Organic is C-H
- Inorganic in non C
- Biochem is living organisms
** Classes of Organic Molecules
*** Carbohydrates
- Disaccharides and polyaccharides
*** Lipids
- Triglycerides
- Phospholipids
- Steroids
*** Proteins
- polypeptides
*** Nucleic Acids
- DNA and RNA
** Organic chemicals
- compounds containing carbon bonded to hydrogens
- carbon is the fundamental element of life
  - 4 atoms in valence
  - single, double, triple covalent bonds
  - linear, branched, or ringed molecules
** Fucntional Groups Continued
- smaller groups of atoms that bind to organic compound
- confer unique reactive properties on the whole molecules
- Hydroxyl (O-H), found in alcohols, and carbohydrates
- Carboxyl (COOH), found in fatty acids, proteins, and organic acids
- Ester (COOR), found in lipids
- Carbonyl (COH), found in aldehydes, polysaccharides
- Phosphate (PO4H2), found in DNA, RNA, ATP
- Methyl (CH3), found in DNA, amino Acids, Lipids, Carbohydrates
** Carbohydrates: Basic Structure
- General Formula (CH2O)
- Basic Structure:
  - Backbone of Carbon
  - Polyhydroxy aldehyde or ketone
- Common Configurations
  - Monosaccharide: polyhydeoxy aldehyde or ketone with 3-7 carbons
  - Disaccharide: two monosaccharides
  - Polysaccharide: five or more monosaccharides
- Changing the chiral orientation of just one C will change the molecule
- Different arrangements will create different structual properties:
  - linear: celluose, structural integriety
  - branched: starch, glycogen, storing energy, easy to pull apart and access
    - you can pull whole branches off of the structure for easy energy access
- Combining two different monosaccharides will form a new carbohydrate
  - table sugar
** How the body uses sugar
- glycogen exists in the body as a resevoir of available energy that is stored in the chemical bonds within individual glucose monomers
- blood sugar
** Lipids
- moleucles composed of mostly hydrogen and carbon
- linked by non polar covalent bonds, they are nonpolar, low solubility in water
  - fatty acids, triglycerides, phospholipids, steroids
  - act as a boundry
- valuable store of energy
- major component of all cellular membranes
- important signaling molecules
*** Fatty acids
- hydrocarbon chain, and a carboxyl group
- all single bonds: saturated fatty acid
- one or more double bonds: unsaturated fatty acids
- >1: polyunsaturated
- 1: monounsaturated
*** Triglycerides
- the majority of the lipides in the body
- glycerol, a three-carbon sugar-alcohol, bonded to three fatty acids
- present in blood and cn be synthesized in the liver
- stored in great quantities in adipose tissue
- energy reserve or the body, during fasting or exercise
*** Phospholipids
- similar in overall sructure to triglycerides, but the third hydroxyl group of glycerol is linked to phosphate
- a small polar (ionized nitrogen-containing molecule) is usually attached to the phosphate
- polar region at one end, two fatty acids make a non-polar region at the opposite end
- they are amphipathic
  - they form lipid bilayers of cellular membranes
*** Steroids
- a distinctly different structure from those of the other subclasses of lipid molecules.
- four interconnected rings of carbon atoms form the skeleton of every steroid
- no water-soluble
- cholesterol, cortisol, estrogen, testosterone
- cholesterol is inserted into the phospholipid bilayer
  - reinforcing the membrane
*** Proteins
- about 50 percent of the organic material in the body (17 percent by weight)
- carbon, hydrogen, oxygen, nitrogen, and small amounts of sulfur
- they are macromolecules, thousands of atoms
- 20 amino acids
- polymer: peptide, polypeptide, protein
[[/home/jackson/Pictures/Screenshot_2022-01-27_09-06-18.png]]
- amino acids are attached through peptides bonds to from proteins
- proteins fold into very specific 3D shapes
- functions: support, enzymes, transport, defense, movement
  - Primary: a series of amino acids bound in a chain. amine acids display small charges functional groups
  - Secondary: develops CO- and NH- groups on adjacent amino acids form hydrogen bonds. This action folds the chain into local configurations called the alpha helic and the beta pleated sheet. Most proteins have both types of secondary structures
  - Tertiary: portions of the secondary structure further interact by forming covalent disulfide bonds and additional interaction. From this emerges a stable three-dimensional molecule. Dependong on the protein, this may be the final function state.
  - Quaternary: Exists only in proteins that consist of more than one polypeptide chain.
- Two variables detrmine the primary structure of a protein
  - the number of amino acids in the chain
  - the specific sequence of different amino acids
** Major Catergories anf Function of Proteins
| Category                               | Functions                                                                                  | Ex                                                                                               |
|----------------------------------------+--------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------|
| <15>                                   | <15>                                                                                       | <15>                                                                                             |
| Proteins that regulate gene expression | make RNA/DNA, make polypeptides from RNA                                                   | transcription factors activate genes; RNA polymerase transcribes genes                           |
| Transporter proteins                   | Mediate the movement of solutes such as ions and organic molecules across plasma membranes | ion channels in plasms membranes allow movement across the memebrane of ions such and Na1 and K1 |
| Enzymes                                | Accelerate the rate of specific chem rxns, such as those required for cellular metabolism  | lipase, amylase, proteases                                                                       |
| Cell Signaling proteins                | Enable cells to communicate with each other, themselbes, and with the external environment | plasma memvrane receptors bind to hormones or neurotransmitters in extracellular fluid           |
| Motor proteins                         | initiate movement                                                                          | myosin, found in muscle cells, contractile force                                                 |
| Structural proteins                    | support, connect, and strengthen cells, tissues, and organs                                | collagen                                                                                         |
| Defense proteins                       | protext against infection and disease                                                      | cytokines and antibodies                                                                         |
** DONE Take notes of 1-4 protein structures
***  Protein Conformation
- in nature proteins appear folded,  bended, or twisted forming more compact strctures
- this is known as a proteins conformation
*** Primary Structure
- primary structure is determined by:
  - the number of amino acids in the chain
  - the specific sequence of different amino acids
- Kinda like a linear chain, just not an exact straight line
*** Secondary Structure
- Attraction between various regions along this linear chain form hydrogen bonds and thus create the secondary structure in a protein
- These are called peptide bonds
- these bonds occur at regular intervals and force the conformation into a spiral or alpha helix
- in addition hydrogen bonds can also form between peptide bonds when extended regions of a polypeptide chain run parallel to each other, forming a relatively straight, extended region known as a beta pleated sheet
- In between these two structures random coil conformations help to link the two together
- these two structures give the protein its ability to anchor itself into a lipid bilayer
*** Tertiary Protein Structure
- after secondary structures are formed additional amino acid side chains become possible
- they fold the polypeptide into three-dimensional conformations,  forming a functional protein
**** Determining Tertiary Structure
1. hydrogen bonds between side groups of amino acids or with surrounding water molecules
2. ionic interactions between ionized regions along the chain
3. interactions between nonpolar regions
4. covalent disulfide bonds linking the sulfur-containing side chains of two cysteine amino acids
5. van der Waals forces
[[/home/jackson/Pictures/fold.png]]
*** Quaternary Protein Structure
- if more than one polypeptide chain is bonded together it is known as a quaternary structure, or multimeric proteins
- the same forces act upon these proteins as described above
- therefore the subnits are held together in the same ways

** Nucleic Acids
Consists of DNA and RNA. Nucleotide monomer.
DNA - deoxyribonucleic acid,  A, T, C, G, double helix, hereditary material
RNA - ribonucleic acid, A, U, C, G, organizes protein synthesis, ribose sugar
Nucleotides are composed of, a phosophate, a pnetose sugar, and a
nintrogenous base (A, T, C, G, or U)
Hydrogen bonds hold DNA together, meaning it is a great way to store information. The 650 million hydrogen bonds in chromosome 1 give it its very strong structure, great for storing information. The half life of DNA can be thousands of years. mRNA has a half life of 30 seconds to 6 hours. This short lived RNA gets the information from DNA to be used somewhere else.
DNA and RNA are negatively charged. A and T will always be together, C and G will always be together, therfore A=T, and C=G. There will always be three rings in the center of the double helix, so one side (A) will have 2 rings, and the other side (T) will have one ring. The number of bonds between A and T will make 2 hydrogen bonds, C and G will form 3. G and C bonds are stronger than the A and T pairing. The reason why hydrogens bonds are better than covalent bonds is because they are weaking, making it much easier to break.
When beginning DNA transcription it is necessary to start in an area with lots of A and T because those bonds are easier to break apart. We use these "markers" to show where to start. Humans have plenty of AT regions.
A pyrimidine and purine combine to give you the three ring structure in the center of the double helix. How DNA is read and replicated is directional. 5' to 3', one strand will be one direction the other will be the other direction, the *anti-parallel* configuration. Both ends will need to be started on opposite ends.
RNA is only a single chain, meaning it can hydrogen bond with itself. Ribose sugar rather than deoxyribose. The pyrimidine base thymine in DNA is replaced by uracil in RNA
** ATP: The Energy Molecule of Cells
Adenosine triphosphate, adenine, ribose, three phosphate.
Function to transfer and storage of energy.
High energy covalent bonds hold the three phosphates together. Three ADHD kids together on one seat in the bus. Kicking out one phosphate releases a ton of energy, puttong that energy into the cell and creating ADP. The bond between phosphate 1-2 is 8 times less energetic than the bond between 2-3. Lots of ATP vs ADP means a high energy state in the cell.
* Chapter 3:  Cell Structure/Function and Metabolic Pathways
| Component              | Structure                                       | Function                                                     |
| Plasma (cell) membrane | Bilayer (phospholipid), embedded proteins       | Cell i/o                                                     |
| Cytoplasm              | Fluid in which organelles float                 | Matrix for rnx                                               |
| Smooth ER              | System interconnected-membrane                  | Metabolisze non-polar,  store ions                           |
| Rough ER               | ""                                              | Assists protein synth                                        |
| Ribosomes              | Granular particle made of protein/RNA           | Synth proteins                                               |
| Golgi complex          | Cluster of flattened sacs                       | Synth carbs, package lipid/glycoproteins > secretion         |
| Mitochondria           | Membranous sac, folded inner parts              | Release energy from food > ATP                               |
| Lysosomes              | Membranous Sac                                  | Digest foreign molecules/old organelles                      |
| Peroxisomes            | Sphere membrane                                 | Hold enzymes to detoxify                                     |
| Centrosomes            | nonmembrane mass, two rods                      | Organize spindle fibers, Chromosome distribution for mitosis |
| Vacuoles               | Membrane sac                                    | Store/Release substances in cytoplasm                        |
| Microfillaments        | Thin hollow tubes                               | support cytoplasm and move moleculed                         |
| Cilia/Flagella         | cytoplasmic projections extend from cell        | move particles outside of cell/move cell                     |
| Nuclear envelope       | bilayer membrane around nucleus (protein/lipid) | nuc support/control passage of materials                     |
| Nucleolus              | nonmembrane from protein and RNA                | Produce rRNA for ribosomes                                   |
| Chromatin              | Fibrous strands composed of protien and DNA     | Hold genetic code, for synth                          |
** Typical Sizes of Cell Structures
Cells are 10 micrometers, cells are small but they are 10-100 times larger than primative cells. We can fit a lot in the cells but they must be organized very well in order to remain functioning. Things are very compartmentalized within cells, in order to allow different chemistry on the inside and outside of memebranes.
** Comparison of Cytoplasm and Cytosol
Cytoplasm: the parts in between the plasma membrane and the nucleus of the cell.
Cytosol: the liquid everything is disolved in, leaving no free space anywhere in the molecule.
** Membranes
The major structural part of the cell. Regulates passage of substances in and out, and between organells and cytosol. Detects chemical messengers at cell surface. Links adjacent cells together by memebrance junctions. Anchors cells to the extracellular matrix.
Some substances see a brick wall, others see a gate. These substances need some form of carrier (usually a protein) to get it. The membrane is a good place for substances to meet up and then to interact.
*** Structure
*Phospholipids* (amphipathic)
Non-polar: 2 fatty acids, tail
Polar: Phosphate head
*Cholesterol*
4 non-polar rings
Interacts strongly with phospholipid fatty acids
We need these two to act somewhere in between a liquid and a solid, creating a dynamic structure for input and output.
This membrane has additional components in order to maintain fluidity.
**** Membrane Proteins
*Integral membrane proteins*: embedded within bilayer, can not be extracted without membrane dusruption, amphipathic, usually reffered to as transmembrane proteins.
*Peripheral membrane proteins*: not amphopathic, located only on membrane surface.
Both are needed in order to have a proper functioning membrane. 1/4 to 1/3 of DNA is for membrane proteins.
These membrane proteins are embedded within or just on the surface.
7tm crosses the bilayer 7 times. Has componenets on both the inside and the outside, allowing a message to bo transmitted seemlessly.
*** Membrance Fluidity
To many unsaturated fatty acids, will be to liquid due to not enough interactions. Leading to leaking cells. To much saturated fatty acids (to much interaction) will not allow anything in or out,  making a solid layer.
Phosopholipids swap approx 10^7 times per second, or move side to side. Introducing cholsterol halps maintain this fluidity.
*** Membrane Junctions
Allow adjacent cells tophysically contact along part of their membrances securely.
*Intergrins*: transmembrane proteins that bind to specific protiens in the extracellular matrix, and link them to membrane proteins on adjacent cells
*Junctions*: desmosomes, tight junctions, gap junctions.
**** Desmosomes
A region between two adjacent cells where adjacent plas, a membranes are seperated by 20 nanometers, dense plaques along the cytoplasmic surface on the plasma membrane. They serve as anchoring points for cadherins, which are proteins that extend from the cell in to the extracellular space, they then link up and bind with cadherins from an adjacent cell. They hold adjacent cells firmly together in areas that are subject to stretching, skin.
**** Tight Junctions
When the extracellular surfaces of two adjacent plasma membranes join, so no space remains between. Occurs in a band around the entire cell. limit movement of material between cells. Gut and bladder.
**** Gap Junctions
Protein channels linking cytosols of 2 cells. Connexins form a small protein-lined channel. Allows small molecules like ions, not allowing larger proteins
** Nucleus
Storage and transmission of genetic information to the next generation of cells
This stored information is usd to synthesize proteins.
Chromatin- DNa in association with protiens forming a fine netwoek of threads
During cell division, chromatin threads become tightly condensed, forming rod-like bodies known as chromosomes.
Surrounding the nucleus is a barrier known as a nuclear envelope, composed 2 mebranes. The membranes are joined together forming nuclear pores, small circular openings. RNA and proteins move thorugh these pores.
Nucleolus: densely staining filamentous region without a membrane prominent in the nucleus. This is the site of ribosome subnit assembly. Otherwise known where ribosomes are made, rRNA and proteins are combines. DNA is coverted to RNA which then folds in on itself.
** Ribosomes
Where proteins are created within a cell, comprised of protein and made using RNA instructions. On ribosomes protein molecules are made from amino acids, using genetic information from mRNA from the DNA in the nucleus.
Small subunit is called 20S, a unit of time 10^-13 seconds. Time it takes for a subunit to reach a no movement in a centrifuge.
60S is the large subunit,
Full ribosome is 80s, within the cytoplasm and attached to somethig in the cell.
2 kinds of ribosomes, free, bound. Free are in cytoplasm making protien. Bound are bound to the ER making a special protein.
** Endoplasmic Reticulum
Rough ER: ribosomes bound to its cytosolic surface, taking RNA and making amino acids language. Proteins go to the golgi apparatus, they must be bound up otherwise they will be off into the cytoplasm.
Smooth ER: no ribosomal particals on its surface, lipids are made, also detocification of hydrophobic molecules. Stores and releases Ca2+ which controls various cells activities such as muscle contraction.
Make more phospholipids, turns cholesterol into other steroids, deals with all sorts of lipids. Removes toxins from the cell, augmenting till its neutralized. Sequesters ions, keeping them out of the cytoplasm, stored for usage.
** Golgi Apparatus
Protein modification, trafficking. The protein stays in the same compartment the entire time, the machine (organell) moves around it. Secretaory proteins are pushed outside the cell. Vesicle proteins
They can also turn into their own organell, something like a digestive enzyme.
The side that faces the ER serves as the site of entry, the other side faces the plasma membrane where products are transfered. While in the sac the molecules is modified, packged, and its container is removed. May create lysosomes, secretory vecisles, or other functions.
** Endosomes
Membrane-bound vesicular and tublar structures that lie between the plasma membrane and the golgi apparatus.
Certain types of vesicles that pinch off the plasma membrane travel to andfuse with endosomes.
Can pinch off vesicles then move to other cell organells or returns to the plasma membrane.
Endosomes are involved in sorting, modifying, and directing vesicular traffic in cells.
** Mitochondira
These organells perform the job of harvesting energy from chemical bonds to create adenosine triphosphate (ATP).
Cellular respiration is the process in which mitochondira form this energy, by consuming oxygen and producing CO2, heat, and water.
Mitochondria have a smooth outer membrane, and a folded inner membrane. This inner membrane is folded into sheets (tubules)known as cristae, which go in to the mictochondria, the inside being called the matrix. Their shape resembles a kidney bean shape with a smooth outer surface and the folded ridge like structure on the inside.
Active cells can contain thousands of mitochondria, less active cells will contain less.
The powerhouse of the cell, most of the ATP that the cell uses is produces here (2 million per second). Electrons are pulled from the H of nutrients. The outer memebrane is very porus, allowing difussion though itself. The inner membrane ins highyly folded (cristae), to increase surface area. Amount of ATP generated is based on surface area. Where the folds go in a proteins complex turns the cristae into an individual energy cell. A battery of sorts, isolating that one cell so that it being dead does not effect others.
Body temperature (37c), within a single cell we have a temperature gradient. Roughly 37, but inside the mitochondira it is 50c (very hot, because it is busy).
*** Unique to Mitochondira
Double membrane, has its own DNA (circular chromosomes/chromatin), has its own dedicated ribosomes (70s), mitochondira was once a bacteria (trace back using DNA sequencing), similar to a gram-negative bacteria (intra-cellular pathogen). Since this bacteria can generate ATP, euchariotic cells kept it around instead of eating it to produce more energy. They became symbiotic becuase the cell got more ATP, and the mitochondira got a safe place to stay, and a steady supply of food.
** Lysosomes
Spherical or oval organelles surrounded by a single memebrane. Cells contain hundreds of lysosomes, each containing acidic fluid and digestive enzymes. Break down bacteria and debris from dead cells. Also break down organelles that have been damaged.
** Peroxisomes
Dense oval bodies with a single membrane. Contain oxygen which helps remove hydrogen from organic molecules (lipds, alcohols, and toxins). Can make and destroy H2O2. Also break down fatty acids, for generating ATP.
** Cytoskeleton
Filament like netowrk associated with processes that maintain and change cell shape and produce cell movements. Some can move and change while some stay still forever. The dynamic ones are micofilimatens and microtubules. Intermediate filaments will stay still.

*** Actin Filaments (microfilaments)
Composed of momomers of the protein G-actin (globular actin). It assembles into a polymer of two chains, known as F-actin (filamentous).
They make up a major portion of the cytoskeleton. Functions include: determining cell shape, abiloty for the cell to move, cell division, and muscle contracting.
*** Intermediate Filaments
Scaffolding on which everything is placed, cell outline, nuclues locating. Composed of different proteins, to be very strong yet flexible. Provide strength to the cells.
*** Microtubules
Interstate system, quite large. Will lie right outside the nucelus the "hub". Centrosome is this hub, composed of two centrioles, composed of nine sets of fused microtubules. Used in mitosis, we need a way to physically seperate chromosomes, the centrosomes acts as anchors for the tug of war seperating of chromosomes. Composed of protein called tubulin. The most rigid part of the strucutre, present in neurons where they provide framework for the cylindrical shape.
*** Cilia
Hair-like extension on the suface of cells. Can generate movement or current of the cell. Respritory movements to trap or move pathogens.
** Proteins
Involved in all physiological processes, from cell signalling to tissue remodeling to organ function. DNA > RNA > Protein (DNA is replicated, Transcription makes RNA, translation goes from RNA to protein (nucleic acid to amino acid using ribosomes)). Called the central dogma of molecular biology. RNA will be destoryed but the proteins will live on.
*** Sequencing Genetic Code
How do we translate nucleic acids to amino acids. By using three letter sequences we create the genetic code. 3 nucletides are called the codon, which is then translated to an amino acid. Some sequences can be punctuation,  61 equal amino acids, the other 3 are stop codons (signaling the end of a sequenece). 1-6 codons can be coded for a single amino acid, they are not random, they always do the same thing.
** DNA Codes
The 64 triples are called the codons, with three of them being the end points. The start codons can also be translated into a amino acid.
** Transcription
AUG starts, UGA is a possible ending.
** mRNA Processing
mRNA is messenger RNA, it goes out into the warzone just after being created. It is single stranded and it is not very strong, and it needs help getting out of the nucleus. So mRNA processing is used to make the message correct, and get it out of the nucleus. Turning it into mature rRNA.
Apply a cap (battle armor) - modified G nucletide
Tail - Poly(A) tail (gotta look fancy), adding lots of a to the end of a sequencd (150-600)
Splicing - Remove introns (removing the non-coding DNA sequences), We dont need all of the message, because some of it is useless. So we remove them so the message is clean.
What is left over is called the Exons, sequences that encode for amino acids.
Splceosoes form the mature mRNA molecules that passed through the nuclear pores to the cytosol. For certain types of tissue some exons may be removed, and they may stay in other tissue types. We always take our the interons, but some jobs may require cetain exons to be removed.
*** Tropomyosin
In muscle you take out 2 4
In smooth mucles you take out  12 13
** tRNA
Two business ends, one does nucleic acid language, the other end does amino acid language. They bring amino acids to the ribosomes for protein synthesis. Anto-codon deals with nucleic acids from mRNA. tRNA have secondary structures allowing them to pair with themselves. They also have a covalent bond between themselves and the amino acid they bring.
Thr tRNA doesnt move in and out, the ribosomes slides along the mRNA to transcribe. There is no amino acid for the stop codon which means there is nothin further to add, so the ribosome slides over and the polypeptide chain is released. Several ribosomes can simultaneously move along a strand of mRNA, producing the same protein in different stages of assembly. As they move along the polypeptide chain gets longer and longer. Called poly ribosomes.
Enzymes presented in the cells cytoplasm called aninoacyl-tRA synthetase enymes are responsible for joining amino acids to the end of tRNA. Each tRNA anti-codon can only bind o one specific amino acid.
*** How does the cell know what to make?
A way to regulated what is being made, and a way to know what they are.
A signal from outside the cell will carry instructions telling it what to produce via a *promoter*. A promoter is a DNA sequence where the enzyme necessary for transcription  is bound. RNA polymerase attaches to this promoter, you also need the DNA to be bent around so that the other binding site on the DNA for transcription can reach, then RNA polymerase races off transcribing the DNA to RNA. The interon is folded in on itself, cut off, and then the exons are reattached.
*** Formation of a Polypeptide
The sequence of the mRNA determines the sequences of the formed polypeptide (amino acids held together by peptide bonds). tRNA and mRNA first bond, the first amino acid detaches fom its tRNA and bonds to the second amino acid, this dipeptide is attached to the second tRNA. This will continue until a stop codon is reached.
While this peptide chain is being formed it starts to experince both secondary and tertiary interactions (twisitng and folding respectivly). This interaction may not occur properly producing toxic complxes, chaperone proteins are there to help prevent this. They help protiens fold corectly as they exit the riosome, they also help different polypeptide chains come together in the proper way to form quaternary structures.
** Mutations
Genes are transscribed into proteins or RNA
By default these genes are turned off, and it takes a lot to turn them on
A permenant change in a nucleotide, adding, or deleting nucleotides from the sequence
This will lead to a change in the structure of the protein
Which may impair cell function or cause cell death
These changes can occur from chemicals or ionizing radiation, and they are random chages. Most of the time we dont even notice them
** Protein Degradation
Proteins degrade at different rates
An unfolded protein is degraded much faster than a folded one
Ubiquitin attached to the protein, moves it to a complex called proteasome, which unfolds the protein and breaks it down into peptides
*** Pathway of proteins destines to be sereated by cells or Transfered to lysosomes
Different sequences will signal to subunits to direct proteins to where they need to go: Secreation, or modification
Membrane protein, Processing or modification, secreted from the cell, Targeted to not leave the cell but not free floating in the cytoplasm
** Terminology
*Ligand* a molecule (can be another protein) or ion that is bound to a protein by one of the following forces:
- electrical attraction between oppositely charged ioni or polarized groupds on the ligand and the protein
- Weaker attractions due to hydrophobic forces between nonpolar regions on the two molecles
This bonding does not involve covalent bonds and are reversible.
*Binding Site* is the region of a protein to which a ligand binds. (think of an enzyme)
A protein can have several binding sites, each specific for a particular ligand.
The binding of a ligand will change the conformation of the protein, the function may change, or it may be activated or deactivated
** Protein Binding: Chemical Specificity
Two proteins must be close enough and in the right conformation to bind. This is known as *chemical specifificy*. They also need to have the proper (opposite) charges in each zone. The fold of the protein will determine what ligand will attach to it, and in what specific regions.
** Protein Binding: Affinity
The strength of the ligand-protein binding
Determines how likely it is that a bound ligand will leave the protein surface and return to its unbound state.
Tight bonds are high-affinity binding sites
Different proteins can bind to the same ligand, with differeing affinities for that ligand
With high affinity very little of that ligand is needed to bind to the protein
Opposite charges, and exact fit create a high-affinity binding site
** Saturation
Equilibrium is reached between unbound ligands in solution and their corresponding protein-binding sites.
*Saturation* refers to the fraction of total binding sites that are occupied at any given time.
Two factors: concentration of unbound ligand in the solution, and the affiniy of the binding site for the ligand
The higher affinity will cause the saturation to increase much more rapidly
** Allosteric Modulation
Occurs when a protein has two binding sites and the noncabalent binding of a ligand to one of the sites alters the shape of the second binding site. Think of it like an *on-off switch*
One binding site on an allosteric protein, known as the *function* site carries out the prtein's physiological funcation.
The other site is the *regulatory site*, the ligand that binds here is called the *modulator molecule*, becuase its binding allosterically modulaed the shape, and therefore the activity of the funcational site
Turns the protein on or off, by modifing it a certain way
** Covalent Modulatin
The covalent bonding of charged chemical groups (phosphates) to some of the protein's side chains. This can changes the activity of the protein. Addition of phosphate is called *phosphorylation*.
The enzymes that mediates protein phosphorylation is called *protein kinase*, the enzyme that mediates the removal of a phosphate is called *phosphoprotein phosphatease*. Both together recognize specific sequences in the protein to find their targets. These move phosphates from the cytoplasm or other parts of the cell to molecules (atp, adp).
** Enzymes and Cellular Energy
Cellular metabolism consists of:
| Catabolism | Breakdown of organic molecules |
| Anabolism  | Synthesis of organic molecules |
Rxns break and form chemical bonds. Hydroxyls perform dehydration, making things bigger. Catabolism involves hydrolysis, breaking bigger things into smaller things. We need to breakdown foods in a "human form" so that they can be used as human building blocks.
** Determinant of Reaction Rates
| Reactant Concentrations | > better        |
| Activation Energy       | < better        |
| Temperature             | > better        |
| Catalyst                | presence better |
** Law of Mass Action
The direction of a chemical reaction is determined (in part) by the concentrations of reactants and products. Reactions like to reach equilibrium, when one side (reactants or products) is higher in concentration it moves to make the other side equal.
** Enzymes
Enzymes are mostly protein molecules, so an enzymes can be defined as a biocatalyst.
They lower *activation energy*, making reactions faster.
They must contact reactants called *substrates*.
The substrate and enzymes beocome bound to each other, forms *Enzyme-substrate complex*, which breaks down to release the product and enzyme.
Enzymes must be left unchaged after the reaction. They help a reaction keep moving in one direction. The substrate is like reactants, but very specific to that enzyme.
*Lock and key*: The substarates fit directly into the rigid enzyme.
*Induced fit*: Slight change in the enzyme to allow then chemical reaction to take place
*** Characterisitcs of Enzymes
| Enzyme not changed                     |
| Are specific for their substrates      |
| Increass reaction rate                 |
| Can cause reversible reactions         |
| Lowers activation energy of a reaction |
*** A specific enzyme catalyzes each cellular reaction
Many enzymes require non-protein helpers called *cofactors* which:
| Bind to the active site |
| Function in catalysis   |
Some are inorganic: ions of zinc, iron, or copper.
Some are organic: Vitamins (*coenzyme*). The B vitamin complex helps speed up many metabolic processes.
*** Regulatin of Enzyme-Mediated Reaction
Enzyme-mediated reactions are regulated by three major factors:
| Substrate Concentration |
| Enzyme Concentration    |
| Enzyme Activity         |
The more substrate present, the faster a enzyme-catalyzed reaction will occur. But reactions cannot exceed the *rate of diffusion*, the rate at which proucts leave, and reactants move in.
Increased enzyme affinity will increase reaction rate.
*** Multiple Sites of binding
An enzyme can have multiple covalent and allosteric activation sites. Decreasing and Increasing enzyme activity, more like a dimmer switch.
For example if you have a lot of ATP you can decrease the acitivty of the enzyme, by actuallying using some ATP. On the other hand ADP can bind to it to maybe increase the speed of reaction making more ATP.
** Metabolic Pathways
Multienzyme pathways are the basis of cellular metabolism.
Example: the transfer of energy released from the breakdown of fuel molecules to ATP.
The products made in one step are the substrate for the next reaction. These are known as *intermediates*.
Since one enzymes cannot perform all of the necessary reaction we need multiple enzymes.
These multienzyme chains can create a feedback loop to stop over-production of a product. This will waste energy if we make too much of one thing. This feedback comes in the form of an *end product inhibitor*, so when enough of the end product is produced it will signal an enzyme further back in the chain to stop (or slow) production of its products. Usually the firt or second enzyme. When all the end-product is used up that enzyme will turn back on an start production. We inibit further back in the pathway so that these first few products can potentially be used elsewhere in the cell. *Negative Feedback Inhibition*.
*** Energy Releasing Pathways
Usually carbohydrates, potentially fats, then proteins. The matabolic pathways that these nutrients go though completly exhaust the entire energy store from a nutrient. They do this slowly as to not induce *combustion*, performing this slowly helps reduce energy loss in the form of heat.
Glycolysis splits glucose, an gives it to the mitochondria.
The mitochondira peforms oxidative phosphorylation moving a phosphate onto ADP to make ATP, using the oxygen.
We can store energy in electrons, so food that has a lot of hydrogens can provide us with a ton of energy. This energy is used to do work.
We pull H of glucose to get energy, and the C gets reduced adding O. Reduced oxygen is also known as water.
** Oxidation-Reduction
Movement of electrons from one molecules to another, a redox reaction
| Loss of electrons is called oxidation     |
| Addition of electrons is called reduction |
| Oxidized = loss of 1 or more electrons    |
| Reduced = gain of 1 or more electrons     |
** Phosphorylation
ATP is formed in glycolysis by *substrate-level phosphorylation* during which:
| An enzyme transfers a phosphate group from a substrate molecules to ADP |
| ATP is formed                                                           |
The compound that forms between the initial reactant, glucose, and the final product, pyruvate are known as intermediates.
The substrate and ADP go into the enzyme, and out comes ATP and the product (minus one phosphate group). This substrate can have 1 or 2 phosphates.
** Glycolytic Pathway
Start with glucose and begin the oxidatin of glucose.
Take hydrogens off of glucose, adding phosphate. Then the cell recoginzizes it as active. ATP is lost as the phosphate is added. Glucose-6-Phosphate is what the cell can use.
